packages/docs/src/page-configs/getting-started/nuxt/code-examples/index.ts

```diff
@@ -1,25 +1,16 @@
-export const pluginCode = `
-import { createVuesticNuxt } from 'vuestic-ui'
-import 'vuestic-ui/css'
-
-export default createVuesticNuxt({
-  config: {
-    // Config here
-  }
-})
-`
-
 export const nuxtConfigCode = `
 export default defineNuxtConfig({
-  alias: {
-    '~normalize.css/normalize.css': 'normalize.css/normalize.css'
-  },
+  buildModules: ['@vuestic-ui/nuxt'],
 
-  meta: {
-    link: [
-      { href: 'https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;1,700&display=swap', rel: 'stylesheet' },
-      { href: 'https://fonts.googleapis.com/icon?family=Material+Icons', rel: 'stylesheet' },
-    ],
-  },
+  vuestic: {
+    config: {
+      // Config here
```

Comment:
Would be nice to have a few words about config. For example, make a reference to `/getting-started/configuration-guide` or `http://localhost:8080/ru/services/global-config`